js-miniapp-sample/package.json

```diff
@@ -17,6 +17,7 @@
     "redux-thunk": "^2.3.0"
   },
   "scripts": {
+    "format": "prettier --config ./prettier.config.js --write 'src/**/*.js'",
```

Comment:
There's a command which looks quite the same, can they be bundled into a single one?